FEM.py

```diff
@@ -0,0 +1,51 @@
+import asyncio
+import random
+import string
+import os
+import time
+
+class FileMonitor:
+    def __init__(self, file1, file2, logfile):
+        self.file1 = file1
+        self.file2 = file2
+        self.logfile = logfile
+        self.count1 = 0
+        self.count2 = 0
+
+    async def random_writer(self):
+        while True:
+            with open(self.file1, 'a') as f1, open(self.file2, 'a') as f2:
+                if random.random() > 0.5:
+                    f1.write("MARUTI ")
+                    self.count1 += 1
+                else:
+                    f1.write(random_string(8) + " ")
```

Comment:
@Mayur1Mtech Why is the string length set to a constant value of 8?

Reply:
It not a constant value...we have random_string(length) function to generate pseudo strings..we can generate string of any length